MSAL/MSAL.xcodeproj/project.pbxproj

```diff
@@ -5230,6 +5298,26 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		28A277CD2C21F0B400D95E00 /* Copy conf.json file */ = {
```

Comment:
Is this expected to be here?

Reply:
Yeah, this is the script used to copy the conf.json